bokehjs/test/playwright.config.ts

```diff
@@ -0,0 +1,11 @@
+import { defineConfig } from '@playwright/test';
```

Comment:
You will have to create `tsconfig.json` file that matches settings of other classes of tests (with any relevant changes) and includes all `*.ts` files added in this PR. Then playwright has to be configured to use that setup somehow. This way we will have consistent type checking setup and it will be possible to lint this code. For linting, you will have to update `.eslintrc.js` accordingly (each `tsconfig.json` needs an entry there).

Reply:
As to linting, when set up properly, you will get all the relevant errors to fix. A quick overview of changes need in this PR is:
1. No `;` unless need to disambiguate code.
2. Trailing commas in all data structures and all other contexts.
3. Snake case instead of camel case or other, unless a name comes from an external source.
4. No unnecessary white space, e.g. in imports.